README.rst

```diff
@@ -243,6 +243,11 @@ Stats Exporter
 .. _threading: https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-threading
 .. _Zipkin: https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-zipkin
 
+Metrics Exporter
```

Comment:
Probably need to put this under "Stats Exporter".

Reply:
Yes. I will change it next time when we move to metrics